anime-list-master.xml

```diff
@@ -15732,6 +15732,7 @@
     <name>Shigofumi</name>
     <mapping-list>
       <mapping anidbseason="0" tvdbseason="1">;1-13;</mapping>
+      <mapping anidbseason="0" tvdbseason="0" start="2" end="8" offset="-1" />
```

Comment:
Got yourself a conflict there with two AniDB episodes (s01 and s02) mapping to the same TVDB episode (s00e01)
```suggestion
      <mapping anidbseason="0" tvdbseason="0" start="1" end="8" offset="-1" />
```

Reply:
s01 is mapped to regular episode 13 (in the row above), so I think it's better to keep start="2" here

Reply:
Oh, right, my bad. I was too focused on the diffs and overlooked that 😅